assignments/closure.js

```diff
@@ -7,23 +7,26 @@ const joke = [];
 
 function dadJoke() {
   joke.push("Why was 6 afraid of 7?");
-
   let setup = [6, 7, 8]
   const punchline = setup.map(x => x + 1);
   joke.push("Because", punchline);
 };
 
-dadJoke()
+dadJoke();
 console.log(joke);
 
-
 /* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
 
-
+var currentCount = 0
```

Comment:
This count variable will be shared between all instances, so you wouldn't be able to create multiple counters or counterFactories.